### ISSUE TITLE
What's the difference between this library and Vsg that is the vulkan osg version?

### ISSUE THREAD

**Issue by happydpc** [pumexx/pumex#7]
I am trying this library with some large models. But the needs are to manage the model in a tree structure  which is similar to OpenSceneGraph.  Picking some objects, hidden some objects. How can I implement these in pumex?

Thank you.

**Comment by pumexx** [comment]
It may be hard to do it in Pumex at the moment and vsg-dev is better suited for the job you are trying to achieve. I still haven't decided on the final outlook of scene graph in the library, because I am researching ways to render it faster.
The problem I have with OSG/VSG is that the scenegraph has to be culled/processed on the CPU side and it may be a bottleneck ( I spent few years working in OSG and it was always the problem with huge scenegraphs ), but at the same time - there are no better ways to deal with it.

**Comment by happydpc** [comment]
Yes, I agree that. I am also working with huge models. And the bottleneck of OSG is very obvious, so I am trying to find some better substitute solution.

**Comment by happydpc** [comment]
Now the Pumex's draw nodes and groups cannot bind to the shader uniform values. This may be the problem. I can't do anything with the group nodes.
